scripts/tag-rc.sh

```diff
@@ -7,5 +7,9 @@ fi
 
 rc_tag="v$(jq -r .version packages/amplify-cli/package.json)"
 commit="$(git rev-parse HEAD~1)"
+short_commit="$(git rev-parse --short HEAD~1)"
 git tag $rc_tag $commit
-git push $rc_tag
+# push the main release tag
+git push origin $rc_tag
+# push the release tags for all other packages
+git tag | grep $short_commit | xargs git push origin
```

Comment:
Do we need to push the git tag for all the other packages?

Reply:
for now yes so that the release tags line up with what's published to npm